Tests/PenguinStructuresTests/FixedSizeArrayTests.swift

```diff
@@ -31,15 +31,16 @@ extension FixedSizeArray {
 }
 
 extension FixedSizeArray where Element: Comparable {
-  func test_MutableCollection<Source: Collection>(_ sortedSource: Source)
-    where Source.Element == Element
+  func test_MutableCollection<C: Collection, D: Collection>(_ source: C, _ distinctSource: D)
```

Comment:
At some point you have to consider generic code "tested," and since each array is just ArrayN<SmallerArray> IMO you don't need to test more than 2 or 3 (max) sizes.